ingest-app/src/migrations/008_add_name_column_bulk_update_status.clj

```diff
@@ -0,0 +1,22 @@
+(ns migrations.008-add-name-column-bulk-update-status
+  (:require
+   [clojure.java.jdbc :as j]
+   [config.migrate-config :as config]))
+
+(defn up
+  "Migrates the database up to version 8."
+  []
+  (println "migrations.008-add-name-column-bulk-update-status up...")
+  (j/db-do-commands (config/db)
+                    "ALTER TABLE CMR_INGEST.bulk_update_task_status ADD
+                     name VARCHAR(255) DEFAULT '' NOT NULL")
+  (j/db-do-commands (config/db)
+                    "UPDATE table SET name = task_id
+                     WHERE name = ''"))
```

Comment:
We should either alter the column definition to DEFAULT to the TASK_ID or add one more alter table change to get rid of the DEFAULT value.

I think the latter is the better choice, so you'd add:
```
(j/db-do-commands (config/db)
                  "ALTER TABLE CMR_INGEST.bulk_update_task_status
                   MODIFY name VARCHAR(255) NOT NULL")
```
as the last step after setting all the values in the table.